src/explore-education-statistics-common/src/modules/table-tool/components/TableHeadersReorderableList.tsx

```diff
@@ -0,0 +1,299 @@
+import DragIcon from '@common/components/DragIcon';
```

Comment:
Looks like there is a couple of unused imports in this file (`useTableHeadersContext` & `useRef`)